notsobot/notsobot.py

```diff
@@ -1562,12 +1647,16 @@ async def flop(self, ctx, urls: ImageFinder = None):
 
             def flop_img(b):
                 with Image.open(b) as img:
-                    img = ImageOps.mirror(img)
+                    image = ImageOps.mirror(img)
+                    img.close()
```

Comment:
This is unnecessary since `img` was created by the context manager which closes it automatically.